eng/common/templates/steps/source-build.yml

```
@@ -97,6 +97,16 @@ steps:
       /p:AssetManifestFileName=$assetManifestFileName
   displayName: Build
 
+- script: $(Build.SourcesDirectory)/test.sh
+  displayName: Run Tests
+- task: PublishTestResults@2
```

The convention in this file is to have blank lines separating each step.

```suggestion

- task: PublishTestResults@2
```

---

Added in https://github.com/dotnet/source-build-reference-packages/pull/824/commits/b2930c5520b6e39c9e578184bbd8cef3186796db:

https://github.com/dotnet/source-build-reference-packages/blob/b2930c5520b6e39c9e578184bbd8cef3186796db/azure-pipelines/builds/ci.yml#L49

Left the steps template untouched due to https://github.com/dotnet/source-build-reference-packages/pull/824#discussion_r1391403747